app/events/email-incoming/index.ts

```diff
@@ -107,7 +107,36 @@ module.exports.handler = createTriggerHandler(
       fromName: 'GCN Circulars',
       recipient: userEmail,
       subject: `Successfully submitted Circular: ${newCircularId}`,
-      body: `Your circular has been successfully submitted. You may view it at ${origin}/circulars/${newCircularId}`,
+      body: `Dear Submitter,
+
+      The email message you submitted to the GCN Circular service has been received and is being distributed to the GCN Circulars subscribers, and posted to the GCN Circulars archive (${origin}/circulars/${newCircularId}). If you have selected to receive Circulars, then you will receive your copy shortly.
```

Comment:
The generic salutation sounds a little spammy. Does it read better without it?

```suggestion
      body: `The email message you submitted to the GCN Circular service has been received and is being distributed to the GCN Circulars subscribers, and posted to the GCN Circulars archive (${origin}/circulars/${newCircularId}). If you have selected to receive Circulars, then you will receive your copy shortly.
```

Reply:
The old email from GCN Classic had that salutation, but I'm fine with not including it.